# Generated for NaoQi method ALLeds.off
# Switch to a minimum intensity a LED or Group of LEDs.

# The name of the LED or Group.
string name
---


